include/riak_kv_vnode.hrl

```diff
@@ -1,83 +1,20 @@
 -include_lib("riak_core/include/riak_core_vnode.hrl").
 
--record(riak_kv_put_req_v1, {
-          bkey :: {binary(),binary()},
-          object :: term(),
-          req_id :: non_neg_integer(),
-          start_time :: non_neg_integer(),
-          options :: list()}).
-
--record(riak_kv_w1c_put_req_v1, {
-    bkey :: {binary(),binary()},
-    encoded_obj :: binary(),
-    type :: primary | fallback
-    % start_time :: non_neg_integer(), Jon to add?
-}).
 
 -record(riak_kv_w1c_put_reply_v1, {
```

Comment:
We left this one as-is because it is a _reply_ record and it would not make sense to mix it into the `riak_kv_requests` module, which (as implied by the name) is intended only for _request_ records. We could, of course, move `riak_kv_w1c_put_reply_v1` into a new module (say `riak_kv_replies`) but since there was only one reply record in existence we decided this was not worth introducing a new module.